cellprofiler/modules/enhanceorsuppressfeatures.py

```diff
@@ -188,7 +188,7 @@ def create_settings(self):
         self.smoothing = cellprofiler.setting.Float(
             'Smoothing scale',
             value=2.0,
-            minval=0,
+            minval=0.1,
```

Comment:
@mcquin What do you think about this default? Do you think epsilon is preferable?

Reply:
Machine epsilon is probably better than `0.1`. 

Alternatively, the smoothing step could be skipped if the value is 0. That would better preserve the current behavior.